docs/Learning-Environment-Design-Agents.md

```diff
@@ -200,29 +200,53 @@ used in your normalization formula.
 
 ### Multiple Visual Observations
 
-Camera observations use rendered textures from one or more cameras in a scene.
-The Brain vectorizes the textures into a 3D Tensor which can be fed into a
-convolutional neural network (CNN). For more information on CNNs, see [this
-guide](http://cs231n.github.io/convolutional-networks/). You can use camera
-observations along side vector observations.
+Visual observations use rendered textures directly or from one or more 
+cameras in a scene. The Brain vectorizes the textures into a 3D Tensor which 
+can be fed into a convolutional neural network (CNN). For more information on 
+CNNs, see [this guide](http://cs231n.github.io/convolutional-networks/). You 
+can use visual observations along side vector observations.
 
-Agents using camera images can capture state of arbitrary complexity and are
-useful when the state is difficult to describe numerically. However, they are
-also typically less efficient and slower to train, and sometimes don't succeed
-at all.  
+Agents using visual observations can capture state of arbitrary complexity and 
+are useful when the state is difficult to describe numerically. However, they 
+are also typically less efficient and slower to train, and sometimes don't 
+succeed at all.
 
-To add a visual observation to an Agent, click on the `Add Camera` button in the
-Agent inspector. Then drag the camera you want to add to the `Camera` field. You
-can have more than one camera attached to an Agent.
+To add a visual observation to an Agent, either click on the `Add Camera` or 
+`Add RenderTexture` button in the Agent inspector. Then drag the camera or 
+render texture you want to add to the `Camera` or `RenderTexture` field. 
+You can have more than one camera or render texture and even use a combination 
+of both attached to an Agent.
 
 ![Agent Camera](images/visual-observation.png)
 
+or
+
+![Agent RenderTexture](images/visual-observation-rendertexture.png)
+
 In addition, make sure that the Agent's Brain expects a visual observation. In
 the Brain inspector, under **Brain Parameters** > **Visual Observations**,
-specify the number of Cameras the Agent is using for its visual observations.
+specify the number of Resolutions the Agent is using for its visual observations.
 For each visual observation, set the width and height of the image (in pixels)
 and whether or not the observation is color or grayscale (when `Black And White`
-is checked).
+is checked). If a combination of `Cameras` and `RenderTextures` is used, all 
+cameras are captured first, then all render textures will be added, both in the
+order they appear in the editor. With 2 cameras and 1 render texture of say twice 
```

Comment:
I think for numbers like this, we need to say "one" and "two" not 1 and 2. 
Also, Why is the render texture using a different resolution ? This is confusing.

Reply:
Reworded this whole section to make it clearer.